EventsExpress/ClientApp/src/components/helpers/Alert.js

```diff
@@ -56,6 +56,12 @@ export default function MySnackbar(props) {
   const { onClose, ...other } = props;
   const { message, open, variant, autoHideDuration } = props.alert;
   const Icon = variantIcon[variant];
+  let timeToShow;
+
+  if(variant !== 'error' && autoHideDuration)
```

Comment:
This should be `variant !== 'error' && !autoHideDuration` because we need to default if there is no specified.